hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js

```diff
@@ -39,7 +39,7 @@ const OrgEntry = () => {
     const { valid, message } = validateEntry(orgName);
     if (valid) {
       const response = await addOrganisation(orgName);
-      if (response && response.data && response.data.status === 'success') {
+      if (response && response.data && response.data.data) {
```

Comment:
Do we really need to check for response and response.data objects? Can we not just check for response.data.data?

Reply:
If we don't and the response body comes back empty then we'll get uninformative errors with cannot access property data of undefined

Reply:
Though if we update to ES2020 there is a more effective way to do this